### PR TITLE
Reconst scripts part4

### PR DESCRIPTION
**Pull request by EmmaRenauld**
# Quick description

Checked the final scripts in our reconst column. Didn't find anything to take out of mains: no new function. Only more comments, fixing prints, etc.

...

## Type of change

Check the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Provide data, screenshots, command line to test (if relevant)

...

# Checklist

- [ ] My code follows the style guidelines of this project (run [autopep8](https://pypi.org/project/autopep8/))
- [ ] I added relevant citations to scripts, modules and functions docstrings and descriptions
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I moved all functions from the script file (except the argparser and main) to scilpy modules
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: scripts/scil_NODDI_priors.py [hunk 1]
```diff
@@ -45,28 +43,29 @@ def _build_arg_parser():
 
     g1 = p.add_argument_group('Metrics options')
     g1.add_argument(
-        '--fa_min', type=float, default='0.7',
+        '--fa_single_fiber', type=float, default=0.7,
```
Comment (karanphil) [inline scripts/scil_NODDI_priors.py:46]: min_fa_single_fiber?

File: scripts/scil_NODDI_priors.py [hunk 2]
```diff
@@ -45,28 +43,29 @@ def _build_arg_parser():
...
         help='Minimal threshold of FA (voxels above that threshold are '
-             'considered in the single fiber mask). [%(default)s]')
+             'considered in \nthe single fiber mask). [%(default)s]')
     g1.add_argument(
-        '--fa_max', type=float, default='0.1',
+        '--fa_ventricles', type=float, default=0.1,
```
Comment (karanphil) [inline scripts/scil_NODDI_priors.py:50]: max_fa_ventricles?

File: scripts/scil_NODDI_priors.py [hunk 3]
```diff
@@ -45,28 +43,29 @@ def _build_arg_parser():
...
         help='Maximal threshold of FA (voxels under that threshold are '
-             'considered in the ventricles). [%(default)s]')
+             'considered in \nthe ventricles). [%(default)s]')
     g1.add_argument(
-        '--md_min', dest='md_min',  type=float, default='0.003',
+        '--md_ventricles', type=float, default=0.003,
```
Comment (karanphil) [inline scripts/scil_NODDI_priors.py:54]: min_md_ventricles?

File: scripts/scil_sh_convert.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 """
-Convert a SH file between the two of the following bases choices:
+Convert a SH file between the two of the following base choices:
```
Comment (karanphil) [inline scripts/scil_sh_convert.py:5]: basis

**Review by karanphil — APPROVED** [review]
Few renaming and it's GTG!

**Review by arnaudbore — COMMENTED** [review]
GTG

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**